src/Microsoft.Health.Fhir.TemplateManagement/Models/ImageInfo.cs

```diff
@@ -73,80 +72,64 @@ public static bool IsValidImageReference(string imageReference)
 
         public static void ValidateImageReference(string imageReference)
         {
```

Comment:
Why we delete reference validation?

Reply:
doesn't delete reference validation, ValidateImageReference() and CreateFromImageReference() both do reference validation, so I delete the code in CreateFromImageReference() and call ValidateImageReference() to do reference validation in CreateFromImageReference().

Reply:
Got it, thanks.